### PR TITLE
Show relative time in history entries (fixes #207)

### LINKED ISSUE #207: Display relative time in history entries
Use a library like *moment.js* to show `3 minutes ago` or `6 months ago` instead `2016-01-13T14:30`

### PR DESCRIPTION
**Pull request by leplatrem**
capture d ecran de 2016-10-04 11-00-19

@n1k0 r?

### CODE REVIEW

File: src/components/HistoryTable.js [hunk 1]
```diff
@@ -22,10 +23,13 @@ class HistoryRow extends Component {
       target,
       user_id,
     } = entry;
+    const tzDate = date + " UTC";
     return (
       <tbody>
         <tr>
-          <td>{date}</td>
+          <td>
+            <span title={tzDate}>{timeago().format(new Date(tzDate))}</span>
```
Comment (n1k0) [inline src/components/HistoryTable.js]: Would be nice to move this to a function in `utils`, wdyt?

File: src/utils.js [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 import type { RecordData } from "./types";
 import React from "react";
+import { default as _timeago } from "timeago.js";
```
Comment (n1k0) [inline src/utils.js]:
```js
import _timeago from "timeago.js";  
```

**Comment by n1k0** [comment]
Flow says:

> src/utils.js:5
> ```
> import { default as _timeago } from "timeago.js";
>                                     ^^^^^^^^^^^^ timeago.js. Required module not found
```

I suspect we have to write the flow type definition for this library. This is to be done in `interface/external_modules`; have a look at how it's done for other external packages we use in the project :)

**Comment by leplatrem** [comment]
Should we use it as well in the records list ?

**Comment by n1k0** [comment]
> Should we use it as well in the records list ?

Yes! Putting the date in a tooltip too :)

**Comment by leplatrem** [comment]
Done

**Comment by n1k0** [comment]
r+

**Pull request merged**